activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -406,12 +426,14 @@ def class_for_adapter(adapter)
           task.is_a?(String) ? task.constantize : task
         end
 
-        def each_current_configuration(environment)
+        def each_current_configuration(environment, spec_name = nil)
           environments = [environment]
           environments << "test" if environment == "development"
 
           environments.each do |env|
             ActiveRecord::Base.configurations.configs_for(env_name: env).each do |db_config|
+              next if spec_name && spec_name != db_config.spec_name
```

Comment:
This means that if you don't pass a `spec_name` and it's `nil` then we just skip creating any databases. That seems like surprising default behavior to me. If it's nil it should create all the databases for that environment which is what the previous behavior was doing.

Reply:
hmm, it is skipped only when `spec_name` is not nil and it's different than `db_config.spec_name`. When it's `nil` it is at it was or am i wrong?
Also all tests are passing.

Reply:
Oh right, I need more ☕️